changelog.d/20240508_111948_abdul.muqadim_is_file_rendered_filter.md

```diff
@@ -0,0 +1 @@
+- [Feature] Introduces the IS_FILE_RENDERED Filter, enabling developers to specify files that should be copied without rendering. (by @Abdul-Muqadim-Arbisoft)
```

Comment:
nit: having a bit more context here can be helpful for the devs.

Reply:
concern addressed